src/features/HelpAndKnowledge/Articles/index.tsx

```diff
@@ -92,14 +93,49 @@ export const covertToLowercaseAndDashes = (string: string) =>
   string.toLowerCase().replace(/\s+/g, '-');
 
 export const ScrollLink = ({ scrollTo }: { scrollTo: string }) => {
+  const scrollToElementAndNavBarOffest = () => {
+    setTimeout(() => {
+      // the height of navigation bar
+      const navBarHeight = document.querySelector(
+        '[data-testid="navigation-bar"]'
+      )?.clientHeight;
+
+      // Find the div to scroll to
+      const scrollElement = document.querySelector(
+        `#${covertToLowercaseAndDashes(scrollTo)}`
+      );
+
+      // Find heading element
+      const headingElement = document.querySelector(
+        `#${covertToLowercaseAndDashes(scrollTo)} > h2`
+      );
+
+      if (!scrollElement || !navBarHeight || !headingElement) {
+        return;
+      }
+
+      // Find the top of the scroll element
+      const { top } = scrollElement.getBoundingClientRect();
+
+      // Find top margin of the heading element
+      const marginOfHeading = parseFloat(
+        window.getComputedStyle(headingElement).marginTop
+      );
+
+      window.scroll(0, top - marginOfHeading - navBarHeight);
+    }, 0);
+  };
+
```

Comment:
Taking much inspiration from the read view scroll function

Reply:
Considering that all of our target anchors are `<h2 />` elements, we can add a class with a css property `scroll-margin-top`.  Those elements are `2rem` plus the `2rem` margin.  We can then give it maybe a little more padding a create a class like so and add it to the anchor elements:

```
.scroll-target {
    scroll-margin-top: 5rem; /* Adjust this value based on the anchor/h2/header height */
}
```

Reply:
Another thought I just had, should I move the covertToLowercaseandDashes to the `h2` element, instead of the div?